runtime/doc/builtin.txt

```diff
@@ -2007,6 +2008,24 @@ complete_info([{what}])				*complete_info()*
 <
 		Return type: dict<any>
 
+complete_match([{lnum}, {col}])			*complete_match()*
+		Returns a List containing the column position and trigger
+		character for completion based on the 'isexpand' option.
+		when 'isexpand' is empty will use the 'iskeyword' option.
```

Comment:
It seems to use the pattern `\k\+$` according to below. So should probably mention that? Or is that too detailed?